doc/source/getting_started/introduction.ipynb

```diff
@@ -0,0 +1,496 @@
+    "\n",
+    "If we are interested in the convex hull of our polygons, we can call `GeoDataFrame.convex_hull()`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gdf[\"convex_hull\"] = gdf.convex_hull"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = gdf[\"convex_hull\"].plot(alpha=.5)  # saving the first plot as an axis and setting alpha (transparency) to 0.5\n",
+    "gdf[\"boundary\"].plot(ax=ax, color=\"white\", linewidth=.5)  # passing the first plot and setting linewitdth to 0.5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Buffer\n",
+    "\n",
+    "In other cases, we may need to buffer the geometry using `GeoDataFrame.buffer()`. Geometry methods are automatically applied to the active geometry, but we can apply them directly to any `GeoSeries` as well. Let's buffer the boroughs and their centroids and plot both on top of each other."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# buffering the active geometry by 10 000 feet (geometry is already in feet)\n",
+    "gdf[\"buffered\"] = gdf.buffer(10000)\n",
+    "\n",
+    "# buffering the centroid geometry by 10 000 feet (geometry is already in feet)\n",
+    "gdf[\"buffered_centroid\"] = gdf[\"centroid\"].buffer(10000)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = gdf[\"buffered\"].plot(alpha=.5)  # saving the first plot as an axis and setting alpha (transparency) to 0.5\n",
+    "gdf[\"buffered_centroid\"].plot(ax=ax, color=\"red\", alpha=.5)  # passing the first plot as an axis to the second\n",
+    "gdf[\"boundary\"].plot(ax=ax, color=\"white\", linewidth=.5)  # passing the first plot and setting linewitdth to 0.5"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "User Guide\n",
+    "    \n",
+    "See more on [geometry creation and manipulation in the User Guide](../docs/user_guide/geometric_manipulations.rst).\n",
+    "</div>\n",
+    "\n",
+    "## Geometry relations\n",
+    "\n",
+    "We can also ask about the relations of different geometries. Using the geometries above, we can check which of the buffered boroughs intersect the original geometry of Brooklyn, i.e., is within 10 000 feet from Brooklyn.\n",
+    "\n",
+    "First, we get a polygon of Brooklyn."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "brooklyn = gdf.loc[\"Brooklyn\", \"geometry\"]\n",
+    "brooklyn"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "type(brooklyn)"
```

Comment:
It might be worth to explicitly call out in the text that we get a Shapely geometry object here, and link to the Shapely docs or so (if that's not yet done earlier on the page)